bakery/src/tasks/upload-book.js

```diff
@@ -38,6 +40,7 @@ const task = ({ awsAccessKeyId, awsSecretAccessKey, bucketName }) => {
           for jsonfile in "$book_dir/"*@*.json; do cp "$jsonfile" "$target_dir/$book_uuid@$book_version:$(basename $jsonfile)"; done;
           for xhtmlfile in "$book_dir/"*@*.xhtml; do cp "$xhtmlfile" "$target_dir/$book_uuid@$book_version:$(basename $xhtmlfile)"; done;
           aws s3 cp --recursive "$target_dir" s3://${bucketName}/contents
+          python /code/scripts/copy-resources-s3.py "$resources_dir" ${bucketName} resources
```

Comment:
```suggestion
          aws s3 cp --recursive "$target_dir" "s3://${bucketName}/contents"
          python /code/scripts/copy-resources-s3.py "$resources_dir" "$bucketName" resources
```
bucketName should be quoted everywhere to prevent arg splitting.